corona-warn-app/cwa-app-android
# Contact Journal Extension - Show RAT and PCR Tests results in Contact Journal (EXPOSUREAPP-6040)

<img width="327" alt="image" src="https://user-images.githubusercontent.com/2919625/119500217-d8b5b400-bd67-11eb-8a4e-cc7b6fb05ff4.png">


### Testing
- Scan PCR/RAT test
- See test results
- If the test is positive or negative new record should be added to DB
- Check Contact Journal to see the corona test record
